app.py

```diff
@@ -27,10 +27,11 @@
 
 import click
 
-from thamos.lib import image_analysis
+from thamos.lib import build_analysis
```

Comment:
Let's wait with merge so that there is PR for `build_analysis` to Thamos.

Reply:
yes, of course this PR depends on 
- https://github.com/thoth-station/common/pull/432
- https://github.com/thoth-station/user-api/pull/490
- https://github.com/thoth-station/thamos/pull/191